api/proto/teleport/usageevents/v1/usageevents.proto

```diff
@@ -505,6 +505,26 @@ enum FeatureRecommendationStatus {
   FEATURE_RECOMMENDATION_STATUS_DONE = 2;
 }
 
+// LicenseLimitEvent is emitted when access to Teleport feature
+// is denied based on license limits
+//
+// PostHost event: tp.license.limit
+message LicenseLimitEvent {
```

Comment:
If this one and `LicenseLimitEvent` from `events.proto` are identical, and even share the same documentation, why don't we just reuse the same proto in both places?

Reply:
That's a good question but I am not sure why it was set up this way. Tentatively tagging @michellescripts or @espadolini who may know more about this.

Reply:
Do we need to emit this from other components in the cluster, or is this always generated in the auth?

Reply:
Currently, it's only emitted from the auth. But I thought of keeping the option open for proxy/webui as well as these components can have richer context on license restriction events.

Reply:
If it's possible to have the event generation be on the auth server, we should. Generating events on agents still implies that partially-trusted agents can emit events that we might get metrics from. I'd remove it from usageevents until we decide that we need it.

Reply:
Noted, I get that using `SubmitUsageEvent` vs. directly using `AnonymizeAndSubmit`.
But I think OPs question is: why is it currently setup that protos be defined in two different places for usageevents instead of reusing them (reuse either from `prehog.v1alpha.teleport.proto` or `usageevents.proto` ) ?

Reply:
TIL. Thank you for explaining 🙏